template-files/stories-templates/pkg.json

```
@@ -0,0 +1,49 @@
+{
+  "scripts": {
+    "start": "expo start --dev-client",
+    "android": "expo run:android",
+    "ios": "expo run:ios",
+    "web": "expo start --web",
+    "eject": "expo eject",
+    "postinstall": "expo-yarn-workspaces postinstall",
+    "stories": "node_modules/expo-stories/bin/index.js start"
```

Can't you provide [`bin`](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#bin) property in `expo-stories/package.json` file and then obtain:
```suggestion
    "stories": "expo-stories start"
```